src/ui/components/shared/Onboarding/DownloadPage.tsx

```diff
@@ -65,9 +63,9 @@ export function DownloadPage({
       </OnboardingBody>
       <DownloadButtons onNext={onNext} />
       <OnboardingActions>
-        <SecondaryLgButton color="blue" onClick={onSkipToLibrary}>
```

Comment:
This is an invalid combination of stuff. Secondary is the red style. Primary is the blue style.